### ISSUE TITLE
Not working in my site

### ISSUE THREAD

**Issue by Jookus** [kessiler/google-adsense#1]
I always get this error: 

Uncaught Error: adsbygoogle.push(): All ins elements in the DOM with class=adsbygoogle already have ads in them.

Is this element still compatible with the newest version of polymer?

**Comment by kessiler** [comment]
Thanks to report!
I will review and update the component.

**Comment by Jookus** [comment]
I think the generated "ins"-node may not be found by the adsense library , because in my website it is created in the shadowDOM.

**Comment by a-haan** [comment]
Still not working after a few tests

**Comment by busynest** [comment]
create a slot within your custom element. then pass your adsense in throught your new tag. derf

**Comment by zricethezav** [comment]
Any update on this?

**Comment by busynest** [comment]
Yes, I figured it out last night but have not got back to it yet.
This wrapper is not needed.
The problem is the custom element itself does not have functionality, because you have not written it in. derf, it's new!

I bet there is a js property you can set to communicate outside of the DOM wall, but I'll get there later.

For now, use the slot, because that is the standard now.

1. add ">slot<" "> /slot<" into your custom element where you want to place your a.d.
2. ">my-element<" Place AdSense Code Here ">/my-element<"

* Your custom element should be loaded from app-shell (upper most container)

*This works because first slot is undefined and loads any content passed into the new tag.
